src/blog/microservices-from-development-to-production.md

```diff
@@ -24,7 +28,7 @@ sbt> install
 
 This "install" command will introspect your project and its sub-projects, generate configuration, package everything up, load it into a local [ConductR](http://conductr.lightbend.com/docs/1.1.x/Home) cluster and then run it all.
 *Just. One. Command.*
-Try doing that with your >insert favourite build tool here<!
+Try doing that with your >insert favourite build tool here<\!
```

Comment:
This is just fixing a syntax warning in my editor (because `<!` is the opening of a HTML doctype or comment). Markdown uses backslash as an escape character. This doesn't affect the rendering, just suppresses the warning.